_constitution/sse_2018_constitution.md

```diff
@@ -5,95 +5,128 @@ title: SSE Constitution
 permalink: about/constitution
 ---
 # Society of Software Engineers Constitution
-_Ratified: July 26, 2016_
+_Ratified: November 2, 2018_
 
 ## DEFINITIONS
+* “Executive board” refers to the President, Vice President, and all Leadership Chairs unless explicitly stated.
+* “Simple majority” refers to greater than 50% of currently assembled members who are able to vote.
+*  “⅔ majority” refers to greater than 66% of currently assembled members who are able to vote.
+
 ## ARTICLE I: Name and Purpose
-### Section 1: Name
-The name of this Organization shall be the Society of Software Engineers, hereafter referred to as the Organization. The official acronym shall be SSE
+###  Section 1: Name
+The name of this Organization shall be the Society of Software Engineers, hereafter referred to as the Organization. The official acronym shall be SSE.
 ### Section 2: Purposes
-The principal purpose of this Organization shall be to improve the experience of students enrolled in the Software Engineering program at the Milwaukee School of Engineering by providing opportunities for collaboration, further education, and representation among other departments at the university. Other goals include increasing enrollment in the program, generating interest in software technology among all students, and guiding new Software Engineering students in integrating with the program.
+The principal purpose of this Organization shall be to improve the experience of students interested in software and affiliated with MSOE by providing opportunities to socialize, network, build their skills, and professionally develop. Other goals include increasing enrollment in corresponding academic programs, generating interest in software technology among all students, guiding prospective students in integrating with the program. Students graduating from the Organization should have developed skills, a broader network, and the confidence to pursue their goals.
```

Comment:
"and" before the "...guiding prospective students in integrating with the program."